logging/log4j1_2/build.gradle

```diff
@@ -33,5 +33,5 @@ projectPomDescription = "This module provides a $project.msftAppInsights appende
 // endregion Publishing properties
 
 dependencies {
-    compile group: 'log4j', name: 'log4j', version: '1.2.17'
+    compileOnly group: 'log4j', name: 'log4j', version: '1.2.17'
 }
```

Comment:
if the tests fail, that probably means you need to pull in log4j as testImplementation (see #2002)